src/main/scala/ch/epfl/bluebrain/nexus/admin/projects/Project.scala

```diff
@@ -5,16 +5,24 @@ import java.util.UUID
 /**
   * Type that represents a project.
   *
-  * @param id           the project internal ID
+  * @param id           the project permanent internal ID
   * @param organization the parent organization ID
   * @param label        the project label (segment)
   * @param description  an optional description
   * @param rev          the resource revision
   * @param deprecated   the deprecation status
   */
-case class Project(id: UUID,
-                   organization: UUID,
-                   label: ProjectLabel,
-                   description: Option[String],
-                   rev: Long,
-                   deprecated: Boolean)
+final case class Project(id: UUID,
+                         organization: UUID,
+                         label: ProjectLabel,
+                         description: Option[String],
+                         rev: Long,
+                         deprecated: Boolean)
+
+/**
+  * Type that holds the actual project payload.
+  *
+  * @param label       the project label
+  * @param description an optional project description
+  */
+final case class SimpleProject(label: String, description: Option[String])
```

Comment:
I'm having difficulties understanding the use of these classes (project, simpleproject, projectlabel), it seems to cause a lot of duplication; could we consider instead using the following?

// assuming ResourceF[A] (notice the lack of the UUID)
```
final case class ResourceF[A](
    id: AbsoluteIri,
    rev: Long,
    deprecated: Boolean,
    types: Set[AbsoluteIri],
    createdAt: Instant,
    createdBy: Identity,
    updatedAt: Instant,
    updatedBy: Identity,
    value: A)
```
the type of projects could be
```
final case class Project[A](id: A, description: Option[String])
```
and we would have project represented by either a `Label`:
```
type ProjectLabel = Project[Label] // just the project label (if that makes sense)
type RootedProjectLabel = Project[(Label, Label)] // includes the org label
type ProjectUUID = Project[UUID]
type RootedProjectUUID = Project[(UUID, UUID)] // includes the org UUID
```
This should allow us to represent all possible project identification schemes an still use `ResourceF[A]` in a disconnected way (i.e.: `ResourceF[Project[Label]]`)